app/components/common/BaseVideo.vue

```diff
@@ -31,6 +32,9 @@ export default {
       TYPES
     }
   },
+  created () {
```

Comment:
did you test it manually? seems like `beforeCreate` maybe better?

Reply:
Ya, I tested it and it worked fine. But beforeCreate would be better, let me test with it.